release-notes/2023-11/README.md

```diff
@@ -1,31 +1,37 @@
 # SystemLink Enterprise release 2023-11 Release Notes
 
-The 2023-11 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+The 2023-11 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
```

Comment:
Probably want to update the release notes template as well: https://github.com/ni/install-systemlink-enterprise/blob/main/chapter-template/release-notes-template.md

Reply:
Good call. I'll include that change here since it's so small.